pkg/finding/finding.go

```diff
@@ -258,6 +214,29 @@ func (f *Finding) moveInputFile(projectDir, seedCorpusDir string) error {
 	return nil
 }
 
+func (f *Finding) ShortDescription() string {
+	// TODO this is just a naive approach to get some error types.
+	// This should be replace as soon as we have a list of the different error types.
+	var errorType string
+	switch f.Type {
+	case ErrorType_CRASH:
+		errorType = strings.ReplaceAll(strings.Split(f.Details, " ")[0], "-", " ")
+	case ErrorType_RUNTIME_ERROR:
+		errorType = strings.Split(f.Details, ":")[0]
+	default:
+		errorType = f.Details
+	}
+
+	output := fmt.Sprintf("[%s] %s", f.Name, errorType)
+
+	// add location (file, function, line)
+	if len(f.StackTrace) > 0 {
+		f := f.StackTrace[0]
+		output = fmt.Sprintf("%s in %s (%s:%d:%d)", output, f.Function, f.SourceFile, f.Line, f.Column)
```

Comment:
There are cases in which in which ASan / libFuzzer don't include the column in the stack trace, so lets do:
```suggestion
		if f.Column != 0 {
			output = fmt.Sprintf("%s in %s (%s:%d:%d)", output, f.Function, f.SourceFile, f.Line, f.Column)
		} else {
			output = fmt.Sprintf("%s in %s (%s:%d)", output, f.Function, f.SourceFile, f.Line)
		}
```

Reply:
ok. I structured the code a little bit different... but should work the same